customer-config/general-ledger-export/gle.sales-and-returns-rules.v1.json

```diff
@@ -18,7 +18,7 @@
             "enum": [
               "ITEM",
               "CATEGORY",
-              "TAX_GROUP",
+              "TAX_GROUP_ID",
```

Comment:
If renaming this to _ID. I guess this should be done for all?
It's also the item id that needs to used for ITEM, so should this then be named ITEM_ID?
And then CATEGORY should probably be named ITEM_CATEGORY_ID